frontend/screens/BrandRegistrationForm.js

```diff
@@ -1,187 +1,206 @@
-            <View style={styles.frameLayout}>
-              <View style={styles.depth4Frame02}>
-                <Text style={[styles.email, styles.emailTypo]}>Brand Type</Text>
-              </View>
-              <View style={styles.depth4Frame1}>
-                <View style={[styles.depth5Frame01, styles.depth5FrameBg]}>
-                  <View style={styles.depth6Frame02}>
-                    <View style={styles.depth7Frame0}>
-                      <Text style={[styles.email1, styles.emailTypo]}>
-                        Brand Type
-                      </Text>
-                    </View>
+          <View style={styles.depth1Frame2}>
+            <View style={[styles.depth2Frame02, styles.frameLayout]}>
+              <View style={styles.frameLayout}>
+                <View style={styles.depth4Frame02}>
+                  <Text style={[styles.email, styles.emailTypo]}>Password</Text>
+                </View>
+                <View style={styles.depth4Frame1}>
+                  <View style={[styles.depth5Frame01, styles.depth5FrameBg]}>
+                    <TextInput
+                      style={styles.textInput}
+                      value={password}
+                      onChangeText={setPassword}
+                      placeholder="Password"
+                      secureTextEntry
+                    />
                   </View>
-                  <Image
-                    style={styles.depth4Frame0}
-                    contentFit="cover"
-                    source={require("../assets/depth-6-frame-12.png")}
-                  />
                 </View>
               </View>
             </View>
           </View>
-        </View>
-        <View style={styles.depth1Frame2}>
-          <View style={[styles.depth2Frame02, styles.frameLayout]}>
-            <View style={styles.frameLayout}>
-              <View style={styles.depth4Frame02}>
-                <Text style={[styles.email, styles.emailTypo]}>Username</Text>
-              </View>
-              <View style={styles.depth4Frame1}>
-                <View style={[styles.depth5Frame01, styles.depth5FrameBg]}>
-                  <View style={styles.depth6Frame03}>
-                    <View style={styles.depth7Frame0}>
-                      <Text style={[styles.email1, styles.emailTypo]}>
-                        Username
-                      </Text>
-                    </View>
+          <View style={styles.depth1Frame2}>
+            <View style={[styles.depth2Frame02, styles.frameLayout]}>
+              <View style={styles.frameLayout}>
+                <View style={styles.depth4Frame02}>
+                  <Text style={[styles.email, styles.emailTypo]}>
+                    Brand Type
+                  </Text>
+                </View>
+                <View style={styles.depth4Frame1}>
+                  <View style={[styles.depth5Frame01, styles.depth5FrameBg]}>
+                    <TextInput
```

Comment:
This should be the a dropDown and we can have below options to start with - 
```
- grocery
- electronics
- fashion 
- toys 
- beauty
- home-decor
- fitness
- education
```

Reply:
This issue includes only intrgation part. Ui enhancement is need for all the pages I had added backend integration. In next Issue I will make this and otp verifications ui enhancement. If you want I will add it in this issue or create one? ui enhancement is needed in 3 screens as the are previously not Inputs just text and images.